index.html

```diff
@@ -4197,16 +4197,20 @@ <h3>Securing Mechanism Specifications</h3>
         </p>
 
         <p>
-Securing mechanism specifications MUST provide a verification mechanism that
-returns only the information in the [=conforming document=] that has been
-secured, without any securing mechanism information included, such as `proof` or
-JOSE/COSE header parameters and signatures. Specifications MAY provide additional mechanisms to convey
-other information that might be helpful (for example, during validation or for
-debugging purposes), such as securing mechanism data. A securing mechanism's
-verification algorithm MUST provide an interface that receives inputs of a media
-type ([=string=] |inputMediaType|) paired with either a sequence of bytes
-([=byte sequence=] |inputBytes|) or a document ([=map=] |inputDocument|), and
-returns a verification result with at least the following [=struct/items=]:
+Securing mechanism specifications MUST provide a verification algorithm that
+returns the information in the [=conforming document=] that has been secured, in
+isolation, without including any securing mechanism information, such as `proof` or
+JOSE/COSE header parameters and signatures. Verification algorithms MAY return
+additional information that might be helpful (for example, during validation or
+for debugging purposes), such as details of the securing mechanism. A verification
+algorithm MUST provide an interface that receives a media type ([=string=]
+|inputMediaType|) and input data ([=byte sequence=] or [=map=] |inputData|).
+If the verification algorithm expects a [=byte sequence=], provide |inputData|
+to the algorithm. If the verification algorithm expects a [=map=], provide the
+result of [=parsing JSON bytes to an Infra value=] given |inputData|. Securing
```

Comment:
I think these instructions are out of place here, because there's no live |inputData| variable in this paragraph. I think this belongs in step 1.2 of https://pr-preview.s3.amazonaws.com/w3c/vc-data-model/pull/1522.html#verification, where you are translating an |inputData| into the parameter type of this interface. Then there's one awkward case: if Verification's caller passed a map, but the securing mechanism expects a byte sequence, we don't have instructions for what to do.

Reply:
Fixed in c90b313fe9d516361088a3cd9b76bd274fafbf8f.